com.streetwriters.notesnook.yml

```diff
@@ -0,0 +1,75 @@
+app-id: com.streetwriters.notesnook
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: start-notesnook
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=system-bus
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+build-options:
+  append-path: /usr/lib/sdk/node16/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: notesnook
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/notesnook/flatpak-node/cache
+        npm_config_cache: /run/build/notesnook/flatpak-node/npm-cache
+        npm_config_offline: "true"
+    build-commands:
+      - ./flatpak-node/electron-builder-arch-args.sh
+      - |
+        cd ./apps/web/desktop
+        cp -r ../build/ ./build/
+
+        # Install npm dependencies
+        npm install --offline
+
+        npm run --offline build
+
+        npx --offline electron-builder $ELECTRON_BUILDER_ARCH_ARGS --linux --dir
+
+        # Bundle app and dependencies
+        cp -r dist/linux-*unpacked/* /app/bin/
+
+      - install -Dm 755 start-notesnook.sh /app/bin/start-notesnook
+    post-install:
+      - install -Dm644 ./apps/web/desktop/assets/icons/16x16.png /app/share/icons/hicolor/16x16/apps/$FLATPAK_ID.png
+      - install -Dm644 ./apps/web/desktop/assets/icons/24x24.png  /app/share/icons/hicolor/24x24/apps/$FLATPAK_ID.png
+      - install -Dm644 ./apps/web/desktop/assets/icons/32x32.png  /app/share/icons/hicolor/32x32/apps/$FLATPAK_ID.png
+      - install -Dm644 ./apps/web/desktop/assets/icons/64x64.png  /app/share/icons/hicolor/64x64/apps/$FLATPAK_ID.png
+      - install -Dm644 ./apps/web/desktop/assets/icons/48x48.png  /app/share/icons/hicolor/48x48/apps/$FLATPAK_ID.png
+      - install -Dm644 ./apps/web/desktop/assets/icons/128x128.png  /app/share/icons/hicolor/128x128/apps/$FLATPAK_ID.png
+      - install -Dm644 ./apps/web/desktop/assets/icons/256x256.png  /app/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+      - install -Dm644 ./apps/web/desktop/assets/icons/512x512.png  /app/share/icons/hicolor/512x512/apps/$FLATPAK_ID.png
+      - install -D com.streetwriters.notesnook.desktop -t  /app/share/applications/
+      - install -D com.streetwriters.notesnook.metainfo.xml -t  /app/share/metainfo/
+    sources:
+      - generated-sources.json
+      - type: git
+        url: https://github.com/streetwriters/notesnook.git
+        branch: master
```

Comment:
```suggestion
```
NEVER git branches.

Reply:
If you see the line below it, it includes a specific git commit hash. From the docs:

> commit (string)
> The commit to use from the git repository. If branch is also specified, then it is verified that the branch/tag is at this specific commit. This is a readable way to document that you're using a particular tag, but verify that it does not change.

Reply:
Done